src/main/java/seedu/address/MainApp.java

```diff
@@ -169,6 +169,7 @@ protected UserPrefs initPrefs(UserPrefsStorage storage) {
     public void start(Stage primaryStage) {
         logger.info("Starting AddressBook " + MainApp.VERSION);
         ui.start(primaryStage);
+        model.deleteExpiredEvent();
```

Comment:
Good that we have this extension now, but we have to wary of potential bugs and also possibly unexpected test failures.